packs/feats/quick-climb.json

```diff
@@ -39,11 +39,15 @@
             },
             {
                 "key": "Note",
+                "outcome": [
+                    "success",
+                    "criticalSuccess"
+                ],
```

Comment:
This is a little complicated, but a creature with a climb speed can climb without rolling and automatically succeed, gaining all benefits that come with it. Usually that's nothing, but this very feat gives an extra five feet of movement.

Reply:
I see what you're saying, but this notes only show up on rolls anyway. We could, in addition to this, append the description of these feats to the Climb and Swim action items via Item Alteration, which would remind players of the feats when they just post the action to chat rather than rolling anything.

Reply:
Eh, better to wait for refactor opportunities in an action overhaul